### ISSUE TITLE
openCamera with mediaType: 'video' freeze

### ISSUE THREAD

**Issue by sulagarde** [ivpusic/react-native-image-crop-picker#959]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.23.0
- react-native v0.57.4

### Platform
Tell us to which platform this issue is related

- iOS

### Expected behaviour
I take a video then I stop register.
The video is taken and I could choose to:
 - retry
 - play
 - use video
I click to 'use video' and the life goes on

### Actual behaviour
I can not click any choice. The screen is freeze.
No error catched.

### Steps to reproduce

1. Open the camera with
ImagePicker.openCamera({mediatype: 'video'}) without openPicker({}) or openCamera({}) first.

**Comment by dhiraj1site** [comment]
Were you able to solve this problem?

**Comment by sulagarde** [comment]
@dhiraj1site no, not yet

**Comment by dhiraj1site** [comment]
The only way I could make it work is by opening a picker first and then opening the video camera. 

```
// can be any media type
if(!this.state.pickerResolve) {
  CropPicker.openPicker({mediaType: "video"})
      .catch(() => {
        this.setState({ pickerResolved: true });
        let video = ImagePickerCrop[selectType]({ mediaType: "video" });
}
else {
        let video = ImagePickerCrop[selectType]({ mediaType: "video" });
}
```

Strange thing is, after any other picker is open first, it works. Might be an easy fix but unfortunately don't have time to work on it.

**Comment by dhiraj1site** [comment]
Yes, it's a bit hacky but you open the picker initially, and if the user cancels the picker, catch that event and open camera, if the user opens any other picker update the state so that you don't do this each time.

**Comment by sulagarde** [comment]
Is there a function/event or something to simulate cancel picker?

**Comment by dhiraj1site** [comment]
I don't think there is https://github.com/ivpusic/react-native-image-crop-picker/issues/291

**Comment by sulagarde** [comment]
Thanks for help :)

**Comment by jaba9531** [comment]
I was experiencing this same behavior on iOS. I fixed it by adding Microphone permissions to Info.plist. This is mentioned in the installation instructions, but is easy to overlook because there is no error output. Hopefully this might help someone who finds this issue!
